framework/source/class/qx/ui/table/model/Filtered.js

```diff
@@ -68,6 +68,10 @@ search.addListener("changeValue", function(e) {
 
 this.getRoot().add(search, {top : 500, left : 10});
 </pre>
+ *
+ * @deprecated {6.0} You should use
+ * <a href='http://developer.mozilla.org/docs/Web/JavaScript/Reference/Objets_globaux/Array/filter' target='_blank'>Array.filter</a>
```

Comment:
@woprandi the "Objet_globaux" part int he URL seems to be for the french link. My link gets rewritten to start with https://developer.mozilla.org/de/... (german) and is suspect @johnspackman s link gets rewritten to https://developer.mozilla.org/en/ . So wie need the full englisch path in the URL